src/tiles/image-block.ts

```diff
@@ -86,12 +86,14 @@ export class ImageBlock extends LitElement {
         display: flex;
         justify-content: center;
         position: relative;
+        background-color: #f1f1f4;
```

Comment:
Q: what color is this? is this special to the tile itself (do we use it anywhere else?)?

Reply:
just for this one